indra/llmessage/llproxy.cpp

```diff
@@ -482,7 +482,7 @@ static apr_status_t tcp_blocking_handshake(LLSocket::ptr_t handle, char * dataou
 
 	apr_size_t expected_len = outlen;
 
-	handle->setBlocking(1000);
+	handle->setBlocking(100000); // 1000ms, 100000us. Should be sufficient for localhost, nearby network
```

Comment:
is this correct?  isn’t 100000us actually equal to 100ms?

Reply:
Fixed the comments, thanks.